enterprise/server/workflow/config/config.go

```diff
@@ -7,26 +7,75 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+// === IMPORTANT ===
+//
+// NOTE: Lines starting with "///" are used to generate markdown documentation
+// for the YAML schema under `docs/workflows-schema.md`.
```

Comment:
EDIT: still looking at other options here -- maybe there's some way to do this with godoc or something else.

Reply:
I'll keep the script as a personal tool for now so that people don't feel like it's a tool they have to learn, and add the comment about manually updating.